ops/storage-ddls/cassandra-setup.cql

```diff
@@ -30,30 +29,28 @@ CREATE TABLE sitesettings(
     cogvisionsvctoken text,
     cogtextsvctoken text,
     insertion_time timestamp,
-    PRIMARY KEY (id, sitename)
+    PRIMARY KEY (sitename)
 );
 
-CREATE INDEX ON sitesettings (sitename);
-
 CREATE TABLE streams (
-  pipeline text,
   streamid uuid,
-  connector text,
+  pipelinekey text,
+  pipelinelabel text,
+  pipelineicon text,
+  streamfactory text,
   params frozen<map<text, text>>,
-  PRIMARY KEY ((pipeline), streamid)
+  PRIMARY KEY (pipelinekey, streamid)
 );
 
 CREATE TABLE trustedsources (
-   sourceid text,
+   externalsourceid text,
    sourcetype text,
-   connector text,
+   streamfactory text,
    rank int,
    insertion_time timestamp,
-   PRIMARY KEY ((connector), sourceid, sourcetype)
+   PRIMARY KEY (pipelinekey, externalsourceid, sourcetype, rank)
```

Comment:
Would it make sense to add the rank as a clustering key not a primary key? In that way, we could order the sources by descending rank which may be useful for display purposes in the dashboard to show the most trusted sources up top.